libs/core/mappers/anime-type.mapper.ts

```diff
@@ -0,0 +1,14 @@
+import { AnimeType } from '../models/anime-type';
+import { AnimeTypeDto } from '../dtos/anime-type.dto';
+
+/** Anime type that mapped to readable condition. */
+export const readableAnimeType = <Readonly<Record<AnimeTypeDto, AnimeType>>>{
```

Comment:
Good approach with enum mapping!

Let's improve the naming here. You are mapping DTO to model, usually, we stick with something like:


```suggestion
export const ANIME_TYPE_MAP_FROM_DTO = Readonly<Record<AnimeTypeDto, AnimeType>>{
```

 We can wrap it in a function to make it a mapper:
 
 ```ts
export namespace AnimeTypeMapper {
  const ANIME_TYPE_MAP_FROM_DTO = Readonly<Record<AnimeTypeDto, AnimeType>> = {/* ... */}

  export function fromDto(dto: AnimeTypeDto): AnimeType {
    return ANIME_TYPE_MAP_FROM_DTO[dto];
  }
}

Reply:
I've tried to make a universal function for making `ANIME_TYPE_MAP_FROM_DTO` but got the error `Property 'TV' does not exist on type 'Record<"Tv" | "Ova" ...`. Maybe there is a better approach for this kind of things?
```js
export const createTypeMapFromDto = <T extends Record<string, string>>(
	sourceDto: T, targetModel: Record<keyof T, string>): Record<keyof T, string> =>
	Object.keys(sourceDto).reduce((acc, key: keyof T) => {
		acc[key] = targetModel[key];
		return acc;
	}, {} as Record<keyof T, string>);
```